commit cb188b0679e409c495404f45379846a36ee3f303
Author: Mark Robert Miller <markrmiller@apache.org>
Date:   Thu Dec 20 15:09:05 2012 +0000

    don't alter recovery wait based on nightly or random multiplier
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/dev/trunk@1424510 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/solr/test-framework/src/java/org/apache/solr/cloud/AbstractDistribZkTestBase.java b/solr/test-framework/src/java/org/apache/solr/cloud/AbstractDistribZkTestBase.java
index 9479da6..5964ca6 100644
--- a/solr/test-framework/src/java/org/apache/solr/cloud/AbstractDistribZkTestBase.java
+++ b/solr/test-framework/src/java/org/apache/solr/cloud/AbstractDistribZkTestBase.java
@@ -122,7 +122,7 @@ public abstract class AbstractDistribZkTestBase extends BaseDistributedSearchTes
   
   protected void waitForRecoveriesToFinish(String collection, ZkStateReader zkStateReader, boolean verbose, boolean failOnTimeout)
       throws Exception {
-    waitForRecoveriesToFinish(collection, zkStateReader, verbose, failOnTimeout, 180 * (TEST_NIGHTLY ? 2 : 1) * RANDOM_MULTIPLIER);
+    waitForRecoveriesToFinish(collection, zkStateReader, verbose, failOnTimeout, 230);
   }
   
   protected void waitForRecoveriesToFinish(String collection,
